### PR TITLE
configurndo pruebas con ts

### PR DESCRIPTION
**Pull request by JuanManu59**

### CODE REVIEW

**Comment by JuanManu59** [comment]
image

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts",
+        "cypress.config.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (danielgalvis98) [inline tsconfig.json]: EOF

File: cypress/e2e/cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  e2e: {
+    setupNodeEvents(on, config) {
+      // modify config values examples
+      // config.defaultCommandTimeout = 10000
+
+      // IMPORTANT return the updated config object
+      return config
+    }
+  }
+})
```
Comment (danielgalvis98) [inline cypress/e2e/cypress.config.ts]: EOF

Comment (JuanManu59) [reply cypress/e2e/cypress.config.ts]: resolved

File: tsconfig.json (hunk 1 above)
Comment (JuanManu59) [reply tsconfig.json]: resolved

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**